system_tests/extnet_tests.py

```diff
@@ -203,6 +204,18 @@ def test_0033_add_ip_range(self):
                 '--ip-range', self._ip_range3])
         self.assertEqual(0, result.exit_code)
 
+    def test_0034_detach_port_group(self):
+        """Detach port group from an external network.
+
+        Invoke the command 'external detach-port-group' in network.
+        """
+        result = self._runner.invoke(
+            external,
+            args=[
+                'detach-port-group', self._name, '--vc-name', ExtNetTest._config['vc2']['vcenter_host_name'],
```

Comment:
Add below line to vcd-cli\base_config.yaml

vc2:
  vcenter_host_name: 'vc2'
  vcenter_host_ip: '<vc2 ip>'
  vcenter_admin_username: 'administrator@vsphere.local'
  vcenter_admin_password: '<vc root password>'